oauthproxy.go

```diff
@@ -307,6 +323,7 @@ func (p *OAuthProxy) buildProxySubrouter(s *mux.Router) {
 	s.Path(signInPath).HandlerFunc(p.SignIn)
 	s.Path(signOutPath).HandlerFunc(p.SignOut)
 	s.Path(oauthStartPath).HandlerFunc(p.OAuthStart)
+	s.Path("/{id}" + oauthStartPath).HandlerFunc(p.OAuthStart)
```

Comment:
Does this mean it's `/foo/oauth2/start`? Would it be better to be `/oauth2/foo/start`?